--- SHA256-0.01/sha384_512.c.orig	2001-03-16 05:35:15.000000000 +0100
+++ SHA256-0.01/sha384_512.c	2025-07-31 19:34:35.468753906 +0200
@@ -16,6 +16,7 @@
  */
 
 #include <stdio.h>
+#include <string.h>
 #include "sha512.h"
 #include "endian.h"
 
--- SHA256-0.01/_sha256.c.orig	2001-03-12 08:14:08.000000000 +0100
+++ SHA256-0.01/_sha256.c	2025-07-31 19:33:52.299951510 +0200
@@ -16,6 +16,7 @@
  */
 
 #include <stdio.h>
+#include <string.h>
 #include "sha.h"
 #include "endian.h"
 
